src/tests/suspense.test.js

```diff
@@ -0,0 +1,33 @@
+import { render, waitForElement, cleanup } from '@testing-library/react'
+import * as React from 'react'
+
+import { useQuery, queryCache } from '../index'
+import { sleep } from './utils'
+
+describe("useQuery's in Suspense mode", () => {
+  afterEach(() => {
+    queryCache.clear()
+    cleanup()
+  })
+
+  it('should not call the queryFn twice when used in Suspense mode', async () => {
```

Comment:
can remove the "when used in Suspense mode" bit, woops.

```suggestion
  it('should not call the queryFn twice', async () => {
```